docs/src/man1/cdist.rst

```diff
@@ -107,6 +107,16 @@ Configure one or more hosts.
 
     Command to use for remote execution (should behave like ssh)
 
+.. option:: -j [JOBS], --jobs [JOBS]
+
+    Specify the maximum number of parallel jobs; currently only
+    global explorers are supported (currently in beta)
+
+.. option:: -b, --enable-beta
```

Comment:
Just a minor suggestion: should we not sort the options by lexicographical order?